### PR TITLE
fix value for combat support, remove unused support_role #37

### PR DESCRIPTION
**Pull request by danielbenedykt**

### CODE REVIEW

File: src/pages/Home/Home.consts.ts [hunk 1]
```diff
@@ -14,8 +14,8 @@ const BUSINESS_VALUES = {
 
 const COMBAT_LEVEL_VALUES = {
   WARRIOR: 'לוחם',
-  COMBAT_SUPPORT: 'תומך_לחימה',
-  SUPPORT_ROLE: 'עורפי',
+  COMBAT_SUPPORT: 'תומך לחימה',
+  //SUPPORT_ROLE: 'עורפי',  // commented, not used
```
Comment (EfratGutman) [inline src/pages/Home/Home.consts.ts:18]: Hey can you please explain why do we remove this option?

Comment (danielbenedykt) [reply src/pages/Home/Home.consts.ts:18]: as far as I can tell,  this  const and text is only defined here and not used in the frontend or backend for any display or logic

Comment (EfratGutman) [reply src/pages/Home/Home.consts.ts:18]: You are right, looks like its not used in the FE.
@bendavidelad @itamargran do you expect to get it in the BE?

Comment (bendavidelad) [reply src/pages/Home/Home.consts.ts:18]: Nope, we only use 'לוחם' or 'תומך לחימה'.
can be removed freely.

**Review by EfratGutman — APPROVED** [review]

**Pull request merged**